src/components/layout/NotificationSnackBar/NotificationSnackBar.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from 'react'
+import { observer } from 'mobx-react'
+import { Snackbar, SnackbarCloseReason, SnackbarContent } from '@mui/material'
+import { NotificationStack } from 'stores/NotificationStore'
+import DonationNotificationLayout from './DonationNotificationLayout'
+
+function NotificationSnackBar({
+  notificationStack,
+}: {
+  notificationStack: NotificationStack | undefined
+}) {
+  const [open, setOpen] = useState(true)
+
+  useEffect(() => {
+    if (notificationStack?.notifications.length) {
+      setOpen(true)
+    }
+  }, [notificationStack?.notifications])
+
+  const handleSnackBarClose = (
+    _event: React.SyntheticEvent | Event,
+    reason: SnackbarCloseReason,
+  ) => {
+    if (reason === 'clickaway') {
+      return
+    }
+
+    setOpen(false)
+    delayOpen()
+  }
+
+  const delayOpen = () => {
+    const interval = setTimeout(() => {
+      notificationStack?.remove()
+      if (notificationStack?.notifications.length) setOpen(true)
+      clearTimeout(interval)
+    }, 2000)
+  }
+
+  return (
+    <Snackbar
+      {...notificationStack?.snackbarProps}
+      open={open && !!notificationStack?.getNotifications.length}
+      onClose={handleSnackBarClose}>
+      <SnackbarContent
+        message={
+          <DonationNotificationLayout
+            data={
+              notificationStack?.getNotifications.length
+                ? notificationStack?.notifications?.[0]
+                : undefined
+            }
+          />
+        }
+        {...notificationStack?.snackbarContentProps}
+      />
+    </Snackbar>
```

Comment:
We already have one `<Snackbar>` for the alerts from the `AlertStore`. There will probably be some problems when showing an alert from both the `<Snackbar>`s

Reply:
Ye this could be a potential problem depending on where those alerts are being shown

Reply:
imho since the two snackbars are shown on different locations there won't be a problem. Also they are just for info and there is no user action expected on them, so even if they overlap or hide each other the user workflow will not be affected.